components/crt_tour/crt_list_tutorial_tip/crt_list_tutorial_tip.tsx

```diff
@@ -14,15 +14,11 @@ import TutorialTip from 'components/tutorial/tutorial_tip';
 import {ShortcutKey, ShortcutKeyVariant} from 'components/shortcut_key';
 import {useMeasurePunchouts} from 'components/tutorial/tutorial_tip/hooks';
 
-type Props = {
-    autoTour: boolean;
```

Comment:
For context, the reason growth team wants to remove autotour is that it did not do as well in a conversion A/B test.

Something I learned today is that growth spike work on separate tour sets still desires to have autotour. Does that sound right, @AshishDhama ? If that's so, I think that's something else we'll want to come to agreement on in the Mattermost use case thread on tour tips.